releasenotes/notes/prepare-0.24-423de82722d2e31a.yaml

```
@@ -0,0 +1,67 @@
+---
+prelude: |
+  This release is a major feature release that includes numerous new features
+  and bugfixes.
+
+  This release is the final release with support for running Qiskit
+  with Python 3.7. Starting in the next minor version release Python >=3.8 will
+  be required to run Qiskit.
+
+  The highlights of this release:
+
+  QuantumInstance, OpFlow, and algorithms usage deprecation
+  ---------------------------------------------------------
+
+  This release officially deprecates the :class:`~QuantumInstance` class (and
+  its associated helper methods and classes), the :mod:`qiskit.opflow` module,
+  and any usage of those in :mod:`qiskit.algorithms`. This deprecation comes
+  from a long thread of work that started in Qiskit Terra 0.21.0 to refactor
+  the :mod:`qiskit.algorithms` module to be based on the computational
+  :mod:`~qiskit.primitives`. There are associated migration guides for any
+  existing users to migrate to the new workflow:
+
+    * ``QuantumInstance`` migration guide: https://qisk.it/qi_migration
+    * ``Opflow`` migration guide: https://qisk.it/qi_migration
+    * Algorithms migration guide: https://qisk.it/algo_migration
+
+  OpenQASM2 improvements
+  ----------------------
+
+  This release includes a major refactoring for the OpenQASM 2.0 support
+  in Qiskit. The first change is the introduction of a new parser for OpenQASM
+  2.0 in the :mod:`qiskit.qasm2` module. This new module replaces the
+  existing :mod:`qiskit.qasm` module. The new parser more explicit and
+  correct relative to the language specification. It is also implemented in
+  Rust and is significantly faster than the previous parser. Paired with the
+  new parser the OpenQASM2.0 exporter underwent a large refactor that
```

Does this miss a whitespace?
```suggestion
  new parser the OpenQASM 2.0 exporter underwent a large refactor that
```